composer.json

```diff
@@ -49,9 +47,6 @@
         "mikey179/vfsstream": "^1.6"
     },
     "extra": {
-        "project-files": [
-            "app/*"
-        ],
```

Comment:
Need to keep this if we're keeping some of the above.

Reply:
Updated